src/server/components/ListOrganizationsNames.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { withTheme } from 'styled-components';
+import { Box, Text } from '@admin-bro/design-system';
+
+const ListOrganizationsNames = (props) => {
+  const token = props.record.params;
+
+  // Filter organizations from params format and delete null values
+  const organizations = [
+    token['organizations.0.name'],
+    token['organizations.1.name'],
+    token['organizations.2.name'],
+    token['organizations.3.name'],
+  ].filter(o => o);
```

Comment:
Can you make this part more dynamic?
now if we add more organizations we should change these part, for instance I added an organization in my DB and it didn't show that to me

<img width="1115" alt="Screen Shot 1401-01-22 at 13 31 05" src="https://user-images.githubusercontent.com/9850545/162738888-059fcf01-8fca-47da-a023-d57b1a771f1b.png">
<img width="847" alt="Screen Shot 1401-01-22 at 13 31 16" src="https://user-images.githubusercontent.com/9850545/162738901-38ca7aa0-a7a9-415c-902e-914d27237546.png">

Reply:
I'll try to make up a solution for the component, but for the form I couldn't pass a async list (or fetched from the db) to the adminbro avaliable values. 
Ill let you know soon.

Reply:
Ok I think I got it for the All Organizations option and the component. 
Missing the permutations list.